controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/ApplicationList.java

```diff
@@ -168,6 +168,8 @@ public ApplicationList allowMajorVersion(int majorVersion) {
                                                 a.deploymentSpec().majorVersion().get().equals(majorVersion)));
     }
 
+    /** Returns the subset of application
```

Comment:
Is this a breaking comment, or a broken github?

Reply:
The former - fixed.